### ISSUE TITLE
IAM issues

### ISSUE THREAD

**Issue by morgante** [terraform-google-modules/terraform-google-project-factory#333]
In `test/setup` across all CFT modules, we are currently seeing issues with adding IAM grants.

Issues look something like this:
```
Error: Batch "iam-project-ci-vpn-6116 modifyIamPolicy" for request "Create IAM Members roles/iam.serviceAccountUser serviceAccount:vpn-int-test@ci-vpn-6116.iam.gserviceaccount.com for \"project \\\"ci-vpn-6116\\\"\"" returned error: Error applying IAM policy for project "ci-vpn-6116": Error setting IAM policy for project "ci-vpn-6116": googleapi: Error 400: Policy members must be of the form "<type>:<value>"., badRequest

  on iam.tf line 32, in resource "google_project_iam_member" "prod_int_test":
  32: resource "google_project_iam_member" "prod_int_test" {
```

This is likely a combination of a problem with Google Provider 2.0's IAM batch handling and the new IAM handling of `deleted:` members. I am investigating further.

**Comment by morgante** [comment]
The immediate quick fix is to pin examples and test/setup to version 2.11.

**Comment by morgante** [comment]
This will be fixed upstream here: https://github.com/terraform-providers/terraform-provider-google/issues/5107

**Comment by morgante** [comment]
This fix is available now in the 2.20.1 version of the provider, we can now pin to `~> 2.20.1`.
